miden-lib/asm/sat/kernel.masm

```diff
@@ -69,26 +100,15 @@ end
 #! - value is the value to increment the nonce by. value can be at most 2^32 - 1 otherwise this
 #!   procedure panics.
 export.incr_account_nonce
-    # AUTHENTICATION
-    # ---------------------------------------------------------------------------------------------
-    # get the hash of the caller
-    padw caller
-    # => [CALLER, value]
-
-    # make sure the caller is a part of the account interface
-    exec.account::authenticate_procedure
-    # => [CALLER, value]
-
-    # drop the caller
-    dropw
+    # authenticate that the procedure invocation originates from the user context
```

Comment:
similar nit to above: maybe this should be "authenticate that the procedure invocations originates from the account context"?